elasticsearch/templates/statefulset.yaml

```diff
@@ -49,7 +49,8 @@ spec:
       schedulerName: "{{ .Values.schedulerName }}"
       {{- end }}
       securityContext:
-        fsGroup: {{ .Values.fsGroup }}
+{{ toYaml .Values.podSecurityContext | indent 8 }}
+        fsGroup: {{ .Values.fsGroup }} # Deprecated value, please use .Values.podSecurityContext.fsGroup
```

Comment:
You will need to wrap this with a `{{ if .Values.fsGroup }}`. Its fine to have the same field twice as the second one is chosen. However this will always set fsGroup to `.Values.fsGroup` when we only want to do it when it is actually defined/set.

Reply:
Oh good point! Changed.